frontend/src/components/messages/components/add-user-form/add-user-form.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { useDispatch } from 'react-redux';
+import { useForm } from 'react-hook-form';
+import clsx from 'clsx';
+import {
+  InputType,
+  InputColor,
+  ButtonType,
+  ButtonColor,
+  ButtonStyleType,
+  UserKey,
+} from 'common/enums';
+import { TextInput, Button } from 'components/common';
+import { MessagesActionCreator } from 'store/slices';
+import { debounce } from 'common/helpers';
```

Comment:
```suggestion
import { debounce } from 'helpers';
```
move helper from `common/helpers` to `helpers`